init.lua

```diff
@@ -112,19 +114,39 @@ local KILL_OK = false
 local AREA_ACTIVATE = KILL_OK
 local AREA_NOACTIVATE = KILL_NO
 
+local savemodeToString = "FALSCH" -- only for debugging
 if safemode then
 	AREA_ACTIVATE = KILL_NO
 	AREA_NOACTIVATE = KILL_OK
+	savemodeToString = "WAHR"
 end
 
 -- Register punchplayer callback.
 minetest.register_on_punchplayer(function(player, hitter, time_from_last_punch, tool_capabilities, dir, damage)
+	local isPlayer = hitter:is_player()
+	
+	if hitter:is_player() == false then
+		return mobsDoNoDamage -- if this is a MOB then give Damage
+	end
+	--[[
```

Comment:
This debug code is commented out; however it should still conform to the existing style. Or, it should be removed.